src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/LanguageServices/TypeInferenceService/AbstractTypeInferenceService.AbstractTypeInferrer.cs

```diff
@@ -79,11 +79,7 @@ private ImmutableArray<TypeInferenceInfo> Filter(IEnumerable<TypeInferenceInfo>
             }
 
             protected IEnumerable<TypeInferenceInfo> CreateResult(SpecialType type, NullableAnnotation nullableAnnotation = NullableAnnotation.None)
-                => CreateResult(Compilation.GetSpecialType(type)
-#if !CODE_STYLE // TODO: remove this #if directive once the below public API is available in CodeStyle layer.
-                        .WithNullableAnnotation(nullableAnnotation)
-#endif
```

Comment:
does these changes impact any tests anywhere?

Reply:
Good question - I did also scan through the conditional directives in tests, but let me double check if something specific to this feature had tests disabled in CodeStyle layer.

Reply:
You are indeed correct - good catch! I did miss unskipping tests that were failing due to this. Thanks!